bedrock_plugin.py

```diff
@@ -30,7 +30,10 @@ def get_conn(self, _headers=None):
 
         session = requests.Session()
         session.headers.update(
-            {"Authorization": f"Bearer {self.bedrock_token}", "Content-Type": "application/json"}
+            {
+                "Authorization": f"Bearer {self.bedrock_token}",
```

Comment:
I'll hesitate using f-strings as Airflow is supported in Python versions 3.5 and lower. For maximizing compatibility, I would suggest str.format()

Reply:
See #10 for fixing this. This diff is just due to applying black on the code.

Reply:
Okay sure